src/pages/profile.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import Button from "@mui/material/Button";
+
+import styles from "../../src/pages/styled";
+
+const Profile = ({ profileInfo, signOut }) => {
+  function createData(title, value) {
+    return { title, value };
+  }
+
+  const rows = [
```

Comment:
Remove it please. Render all the expected fields without this constant by using profileInfo